### DISCUSSION TITLE
Doctor Who Gallifreyan Style Clock webpage wallpaper

### DISCUSSION THREAD

**Discussion by meWho3000** [rocksdanister/lively#1497] [Show and tell]
Just finished making this single webpage of animated Gallifreyan Rings, works well with Lively as a **Doctor Who theme wallpaper**. The clock elements are built in. Click on the lower right question mark on the page to see each ring's function. Feedbacks are welcome🙂 Enjoy💙
https://www.mewho.com/doctorwho

![dw1](https://user-images.githubusercontent.com/82191558/208291260-a635807a-39b0-4d40-9a7c-f0a403074931.jpg)
![dw6](https://user-images.githubusercontent.com/82191558/208291664-eb456da1-d5e4-45e2-ac0d-ed297f84a361.jpg)
![dw4](https://user-images.githubusercontent.com/82191558/208291666-10fe0f5f-a702-4fff-90cc-534a6e9ad920.jpg)

**Comment by rocksdanister** [comment]
Looks nice, if you make it offline maybe have support for Lively API (customisation and music reaction):
https://github.com/rocksdanister/lively/wiki/Web-Guide-IV-:-Interaction
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data

It also makes it easy for me to share in Gallery once released.

**Reply by meWho3000** [reply]
oh wow, I didn't realize that Lively API can work with local webpages. Cool & thx! I will definitely look into it.
The JWST space images are href links to photos from NASA's Flickr album. Will the links work or will I need to have the JPEGs stored locally?

**Reply by rocksdanister** [reply]
local would be better because sometimes internet may not be available, but online will also work.

**Reply by meWho3000** [reply]
Got it. Thanks & thanks for making Lively!!
